src/components/pages/docs.yml

```diff
@@ -16,3 +16,6 @@ initialAuthor:
 filterPlaceholder:
   en: Filter by title or initial author
   de: Nach Titel oder Erstautor filtern
+badges:
+  en: Badges
+  de: Abzeichen
```

Comment:
Although this is the main translation, it reminds me too much of a piece of metal on clothes. Maybe we could use "Kennzeichen" instead?

Reply:
"Kennzeichen" it is then 👍